site/src/pages/ManagementSettingsPage/IdpSyncPage/IdpSyncPageView.tsx

```diff
@@ -19,22 +23,57 @@ import {
 	TableLoaderSkeleton,
 	TableRowSkeleton,
 } from "components/TableLoader/TableLoader";
+import { TabLink, Tabs, TabsList } from "components/Tabs/Tabs";
 import type { FC } from "react";
+import { useSearchParams } from "react-router-dom";
 import { MONOSPACE_FONT_FAMILY } from "theme/constants";
 import { docs } from "utils/docs";
+import { ExportPolicyButton } from "./ExportPolicyButton";
+import { IdpPillList } from "./IdpPillList";
 
-export type IdpSyncPageViewProps = {
-	oidcConfig: OIDCConfig | undefined;
-};
+interface IdpSyncPageViewProps {
+	groupSyncSettings: GroupSyncSettings | undefined;
+	roleSyncSettings: RoleSyncSettings | undefined;
+	groups: Group[] | undefined;
+	organization: Organization;
+}
+
+export const IdpSyncPageView: FC<IdpSyncPageViewProps> = ({
```

Comment:
Have to put the comment here because the code was already in place previously, and it's not selectable for review:

I think we need to add some CSS optical adjustments to the key-value pairs for the `IdpField` component. Since we're using two different typefaces, their baselines are slightly different. And when you place them right next to each other on the same line, there's enough variation that the text looks sloppy

You might have to open this in a new tab, but the baselines are ever so slightly off, and the cap-heights are, too:
<img width="1104" alt="Screenshot 2024-09-23 at 12 21 28 PM" src="https://github.com/user-attachments/assets/f9f8b8ca-4e2e-4380-86da-a50057bd9616">

Reply:
Though a slightly easier and probably much more manageable alternative would be to set all the text in monospace

Reply:
The monospace font was supposed to be a temporary measure until this page becomes a form and these would become fields. I'll see if I can improve it for now.